### DISCUSSION TITLE
通过 gRPC 传递 Request ID

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#404] [Announcements]
# 通过 gRPC 传递 Request ID

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2019/03/10/request-id-in-grpc/

**Comment by repairbean** [comment]
Java版本的grpc传递traceId，好像没什么特别合适的开源方案呀

**Reply by Xuanwo** [reply]
不懂 Java，但是原理应该差不多？
